src/utils/helpers/endpoints/types.ts

```diff
@@ -40,7 +40,7 @@ export type CheckUniqueEndpoint = [
         table: string
         field: string
       }
-  ) & { value: string }
+  ) & { value: string; caseInsensitive?: boolean }
```

Comment:
Makes type consistent with change done by @fergie-nz in https://github.com/openmsupply/conforma-server/pull/810